packages/@lwc/compiler/src/transformers/__tests__/transform-css.spec.ts

```diff
@@ -137,7 +106,7 @@ describe('regressions', () => {
         const actual = `.foo { content: "x\\x"; }`;
         const expected = `
             function stylesheet(hostSelector, shadowSelector, nativeShadow) {
-                return [".foo", shadowSelector, " {content: \\"x\\\\x\\";}\\n"].join('');
```

Comment:
This can be removed, it's a duplicate of `packages/@lwc/style-compiler/src/__tests__/fixtures/back-slash/actual.css`.